main/src/cgeo/geocaching/connector/su/package-info.java

```diff
@@ -1,4 +1,4 @@
 /**
- * Russian <a href="http://www.geocaching.su">Geocaching.su</a> implementation.
+ * Russian <a href="https://www.geocaching.su">Geocaching.su</a> implementation.
```

Comment:
a bit of off-topic, but `www` is also obsolete now and can be removed :)

Reply:
Will change it.